brainscore_language/benchmarks/pereira2018_v2022/__init__.py

```diff
@@ -0,0 +1,135 @@
+import logging
+
+from brainio.assemblies import NeuroidAssembly
+from brainscore_core.benchmarks import BenchmarkBase
+from brainscore_core.metrics import Score
+from brainscore_language import load_dataset, load_metric, benchmark_registry
+from brainscore_language.artificial_subject import ArtificialSubject
+from brainscore_language.utils.ceiling import ceiling_normalize
+from brainscore_language.utils.s3 import load_from_s3
+
+logger = logging.getLogger(__name__)
+
+BIBTEX = """@article{pereira2018toward,
+  title={Toward a universal decoder of linguistic meaning from brain activation},
+  author={Pereira, Francisco and Lou, Bin and Pritchett, Brianna and Ritter, Samuel and Gershman, Samuel J 
+          and Kanwisher, Nancy and Botvinick, Matthew and Fedorenko, Evelina},
+  journal={Nature communications},
+  volume={9},
+  number={1},
+  pages={1--13},
+  year={2018},
+  publisher={Nature Publishing Group}
+}"""
```

Comment:
@kvfairchild we're beginning to duplicate this `BIBTEX` quite a lot, e.g. in [`data/pereira2018`](https://github.com/brain-score/language/blob/b03ea2c9ceef646c53a7caedcab9ac13df07354a/brainscore_language/data/pereira2018/__init__.py#L8-L18) and [`benchmarks/pereira2018`](https://github.com/brain-score/language/blob/b03ea2c9ceef646c53a7caedcab9ac13df07354a/brainscore_language/benchmarks/pereira2018/__init__.py#L13-L23). One simple solution might be to replace the `benchmarks` definitions with `from brainscore_language.data.pereira2018 import BIBTEX`, is that acceptable from a plugin perspective? I guess we might run into import errors if dependencies are not installed and potentially encourage direct imports when we want plugins to go through the plugin manager.